configs/experimental/windows/0.1.0/bazel_experimental/BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Bazel Authors. All rights reserved.
+# Copyright 2018 The Bazel Authors. All rights reserved.
```

Comment:
I think this should be kept as 2016. Last time, Damien mentioned it's the year when Bazel was first released.

Reply:
Done.